src/olympia/activity/tests/test_commands.py

```diff
@@ -8,7 +8,7 @@
 
 class TestRepudiateActivityLogToken(TestCase):
     def setUp(self):
-        self.version = addon_factory().latest_version
+        self.version = addon_factory().current_version
         self.token1 = ActivityLogToken.objects.create(
```

Comment:
Gotcha - I was lazy and went for the quick fix, but you're right, it makes more sense that way.